### PR TITLE
KAN-6: Delete todo item

### PR DESCRIPTION
**Pull request by KupoKopu**
Jira ticket:
https://ada-team-lukas.atlassian.net/browse/KAN-6

Now can delete a todo item from a delete button on todo item row.
Deletes using the todo's ID

Before Delete
image

After Delete
image

Coverage report
image

Lighthouse
image

### CODE REVIEW

File: app/templates/index.html [hunk 1]
```diff
@@ -8,6 +8,9 @@
             <p>{{ todo.task }}</p>
             <br>
             {{ todo.description }}
+            <form action="{{ url_for('main.delete_todo', todo_id=todo.id) }}" method="POST" style="display:inline;">
+                <button type="submit" class="btn btn-danger">Delete</button>
```
Comment (luyah231) [inline app/templates/index.html]: button needs to be aligned to the right

**Comment by luyah231** [comment]
button should also match design and be more of an outline style

File: app/templates/index.html (hunk 1 above)
Comment (luyah231) [inline app/templates/index.html]: button should be be more of outline style and not red

Comment (KupoKopu) [reply app/templates/index.html:12]: I think red is better since it hints risk in role of the button to delete

Comment (KupoKopu) [reply app/templates/index.html]: I think red is better since it hints risk in role of the button to delete

Comment (luyah231) [reply app/templates/index.html]: cool, keep it like that but update the style in the mock up

**Review by luyah231 — APPROVED** [review]

**Pull request merged**